### PR TITLE
I added seven migrations to the database

### PR DESCRIPTION
**Pull request by Magi1610**
Added 7 new database migrations 
student migration to manage student data
loan migration to track loan records
return migration to handle return processes
category migration to categorize materials
material migration to store material information
Inventory Migration to manage inventory
Loan Detail Migration to store loan detail

### CODE REVIEW

File: database/migrations/2024_09_17_210612_create_student_table.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('student', function (Blueprint $table) {
+            $table->id();
+            $table->string('studentName');
+            $table->string('paternalSurname_student');
+            $table->string('maternalSurname_student');
```
Comment (temolzin) [inline database/migrations/2024_09_17_210612_create_student_table.php]: last_name

File: database/migrations/2024_09_17_211414_create_loans_table.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('loans', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('student_id');
+            $table->datetime('loan_datetime');
```
Comment (josejulianmtnez) [inline database/migrations/2024_09_17_211414_create_loans_table.php]:
```suggestion
            $table->timestamp('loan_datetime');
```

File: database/migrations/2024_09_17_211414_create_loans_table.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            $table->datetime('return_datetime');
```
Comment (josejulianmtnez) [inline database/migrations/2024_09_17_211414_create_loans_table.php]:
```suggestion
            $table->timestamp('return_datetime');
```

File: database/migrations/2024_09_17_213434_create_material_returns_table.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('material_returns', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('student_id');
+            $table->unsignedBigInteger('loan_id');
+            $table->datetime('delivery_datetime');
```
Comment (josejulianmtnez) [inline database/migrations/2024_09_17_213434_create_material_returns_table.php]: Procura usar timestamp en lugar de datetime (no confundir con timestamps)

File: database/migrations/2024_09_17_215828_create_inventories_table.php [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('inventories', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('material_id');
+            $table->integer('total_quantity');
+            $table->integer('loaned_quantity');
+            $table->dateTime('registration_datetime');
```
Comment (josejulianmtnez) [inline database/migrations/2024_09_17_215828_create_inventories_table.php]:
```suggestion
            $table->timestamp('registration_datetime');
```

**Pull request merged**